settings.json.template

```diff
@@ -409,6 +409,25 @@
   "indentationOnNewLine": false,
   */
 
+  /*
```

Comment:
These modifications will have to be ported to settings.json.docker, too (they have to be the same file). I can do it.

Do we want to expose the rate limiting config knobs via an environment variable under docker? I think it may make sense. Agreed?

Reply:
I literally know nothing about ANY container platforms so I have no idea!  The docker guys would know :)

Reply:
Ok I'll do it.

Reply:
Done. I'll introduce environment variables if we decide to uncomment the rate limiting configurations in the next thread.